backend-tests/tests/test_inventory.py

```diff
@@ -277,13 +277,13 @@ def test_ok(self, user):
 
             api_dev = r.json()
             # Expected inventory count per scope:
-            # {"inventory": 3, "identity": 1, "system": 2}
-            assert len(api_dev["attributes"]) == 6
+            # {"inventory": 3, "identity": 1+2, "system": 2} # +2 comes from the id_data see MEN-3637
+            assert len(api_dev["attributes"]) == 8
             # new scopes: identity and system holding authset status and
             #             time-stamp values respectively
 
             for a in api_dev["attributes"]:
-                if a["name"] == "mac":
+                if a["name"] == "mac" && a["scope"] == "inventory":
```

Comment:
Mmm, then we are not checking any more `mac` for other scopes? Also, this change is silent (i.e. no mention in the commit) so I don't really understand the context.

Reply:
@merlin-northern Thank you for a beautiful commit message :heart: